popin-action-page.php

```diff
@@ -0,0 +1,19 @@
+<?php get_header(); ?>
+
+	<div class='main-section'>
+	  <ul class="posts">
+		<?php 
+		  if ( have_posts() ) :
+	      	while ( have_posts() ) : the_post();
```

Comment:
Indentation. But it also looks like a general page. Not sure where you're going with this file or when this should be used.